helm/templates/minio-external-scaler-scaledobject.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if .Values.minio.scaledobject.enabled }}
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
```

Comment:
the configuration of the scaled object should be configurable from the helm values